packages/models-library/src/models_library/utils/change_case.py

```diff
@@ -7,10 +7,11 @@
     class Config:
         extra = Extra.forbid
         alias_generator = snake_to_camel  # <--------
-        json_loads = orjson.loads
+        json_loads = json_loads
```

Comment:
Q: is this correct?

Reply:
@GitHK check the implementation of `json_loads` :-)